app/templates/components/notification-dropdown.hbs

```diff
@@ -10,20 +10,22 @@
       {{/link-to}}
     </div>
     <div class="ui basic right aligned segment weight-400">
-      <a href="#">{{t 'Mark all as Read'}}</a>
+      <a href="#" {{action 'markAllRead'}}>{{t 'Mark all as Read'}}</a>
     </div>
   </div>
   <div class="ui fluid link celled selection list">
     {{#each notifications as |notification|}}
```

Comment:
@niranjan94 So shall I restore the previous version in which we show only the unread notifications in dropdown? Or keep this new one in which we show all the notifications like facebook and differentiate the read and unread notifications. Also clicking on any notification will mark it as read. Opinions?